diagrams/claim-example-2.svg

```diff
@@ -6,45 +6,24 @@ A subject, called "Pat", has a property "dateOfBirth", with the value
 "2010-01-01". The intended meaning is that Pat was born on 1 January 2010.
     </desc>
     <defs>
-        <style>
-            rect,
-            ellipse {
-                stroke-width: 1;
-                stroke: #000
-            }
-
-            .literal {
-                font-family: serif
-            }
-
-            line {
-                stroke: #000;
-                stroke-width: 2;
-                marker-end: url(#arrow)
-            }
-
-            text {
-                fill: #000;
-                font-size: 10px;
-                font-family: sans-serif
-            }
-
-            .background {
-                stroke: none;
-                fill: #fff
-            }
-        </style>
-        <marker id="arrow" viewBox="0 0 6 4" refX="4" refY="2" fill="#000"
-            markerWidth="6" markerHeight="4" orient="auto">
-            <path d="M0 0v4l6 -2z" />
-        </marker>
+      <style><![CDATA[
+          rect, ellipse {stroke-width: 1;stroke: #000}
+          .literal {font-family: serif}
+          line {stroke: #000;stroke-width: 2;marker-end: url(#arrow)}
+          text {fill: #000;font-size: 10px;font-family: sans-serif}
+          #background {stroke: none;fill: #fff;width:100%;height:100%}
+      ]]></style>
+      <marker id="arrow" viewBox="0 0 6 4" refX="4" refY="2" fill="#000"
```

Comment:
Is this fill value correct as only 3 hex characters? The GH review is highlighting this which makes me think it needs to be 6 but I'm not an expert in SVGs.

Reply:
yes it's correct.

(specifically SVG converts three-character values to six by doubling up each character, so 000 -> 000000 and 4f8 -> 44ff88 etc)

Reply:
Ok good to know. Thanks for the insight!